src/mjolnir/landmarks.cc

```diff
@@ -323,8 +323,9 @@ bool BuildLandmarkFromPBF(const boost::property_tree::ptree& pt,
   }
 
   LOG_INFO("Parsing nodes and storing landmarks...");
+  OSMPBF::Parser parser(4);
```

Comment:
Better to use something like `"mjolnir.concurrency"` or `std::thread::hardware_concurrency()` instead of hardcoded `4`, e.g. like here: https://github.com/valhalla/valhalla/blob/58a289de51638c55f479101bf5352160902e0182/src/mjolnir/graphbuilder.cc#L1473-L1475